pkg/test/harness.go

```diff
@@ -393,6 +402,16 @@ func (h *Harness) Stop() {
 		}
 	}
 
+	if h.bgProcesses != nil {
+		for _, p := range h.bgProcesses {
+			h.T.Log("killing process ", p)
+			err := p.Process.Kill()
+			if err != nil {
+				h.T.Log("background process kill error", err)
+			}
+		}
+	}
+
```

Comment:
This needed to move up... the case below (which previously came before could return) leaking back processes